azkaban-common/src/main/java/azkaban/imagemgmt/daos/ImageTypeDao.java

```diff
@@ -86,4 +86,21 @@
    * @return ImageType
    */
   public ImageType getImageTypeWithOwnershipsById(String id);
+
+  /**
+   * Add an owner for a given image type.
+   *
+   * @param imageType - ImageType to be updated
+   * @return int - Id representing image type metadata
+   */
+  public int addOwnerOfImageType(ImageType imageType) throws ImageMgmtException;
```

Comment:
how about using addImageTypeOwner as the function name? the same for the method below.

Reply:
ack